Endpoint management RBAC (#3040)
elastic/security-docs: docs/getting-started/defend-feature-privs.asciidoc

NOTE: {elastic-defend}'s feature privileges must be assigned to *All Spaces*. You can't assign them to an individual {kib} space. 

[role="screenshot"]
image::images/endpoint-privileges.png[Configuring privileges in Kibana,75%]

To grant access, select *All* for the *Security* feature in the *{kib} privileges* configuration UI, then turn on the *Customize sub-feature privileges* switch. For each of the following sub-feature privileges, select the type of access you want to allow:

* `All`: Users have full access to the feature and can perform all available actions and configuration.

> * `Read`: Users can view the feature, but they can't perform any actions or configuration. (Not all privileges have this option.)

Reviewer: ```suggestion
* `Read`: Users can view the feature, but they can't perform any actions or configuration. (Some sub-features don't have this privilege option.)
```

* `None`: Users can't access or view the feature.

[cols="1,1",width="100%"]
|==============================================
| *Endpoint List*
| Access the <<admin-page-ov,Endpoints>> page, listing all hosts running {elastic-defend} and their relevant integration details.

| *Trusted Applications*

> | Access the <<trusted-apps-ov,Trusted Applications>> page to mitigate conflicts with other software, such as other antivirus or endpoint security applications.

Reviewer 2: ```suggestion
| Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as other antivirus or endpoint security applications.
```

NOTE: {elastic-defend}'s feature privileges must be assigned to *All Spaces*. You can't assign them to an individual {kib} space. 

[role="screenshot"]
image::images/endpoint-privileges.png[Configuring privileges in Kibana,75%]

To grant access, select *All* for the *Security* feature in the *{kib} privileges* configuration UI, then turn on the *Customize sub-feature privileges* switch. For each of the following sub-feature privileges, select the type of access you want to allow:

> * *All*: Users have full access to the feature and can perform all available actions and configuration.

Reviewer 3: `configuration` sounds like a verb here, so adding one before. 
```suggestion
* *All*: Users have full access to the feature, which includes performing all available actions and managing configuration.
```

* *Read*: Users can view the feature, but they can't perform any actions or configuration. (Some features don't have this privilege.)
* *None*: Users can't access or view the feature.

[cols="1,1",width="100%"]
|==============================================
| *Endpoint List*
| Access the <<admin-page-ov,Endpoints>> page, which lists all hosts running {elastic-defend}, and associated integration details.

| *Trusted Applications*

> | Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as other antivirus or endpoint security applications.

Reviewer 3: ```suggestion
| Access the <<trusted-apps-ov,Trusted Applications>> page to remediate conflicts with other software, such as antivirus or endpoint security applications.
```
